german/Stopfen/stopfen.md

```diff
@@ -8,7 +8,7 @@ die Ruder?! Sie stellen fest, dass die Paddel nicht im Boot sind.
 
 Was machen Sie nun?
 
-[Im Boot nach was suchen was als Paddel gebraucht werden kann] (../suchen/suchen.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.